### ISSUE TITLE
Note: Perceptual hash and cryptographic hashes are different

### ISSUE THREAD

**Issue by DonaldTsang** [ledinhtri97/Hadoop-Image-Matching#2]
Let me guess what is in this repo: Are you using skein to fuzzy match images?

**Comment by ledinhtri97** [comment]
Nope, I'm using `ahash`. Sorry about topic, maybe made you're confused, but when using `skein` to finding match images, it's not cover the field that image have smaller/bigger size (cropped), difference color mode, or have difference type image (jpg, png, jpeg,...) etc,... . So `ahash` help me to solver that problem.

**Comment by DonaldTsang** [comment]
Understood (aHash is pretty good in use if wHash is not available), also in that case skein is just to prevent file duplication? IDK might want to be more detailed about what skein is used for.
